content/rest-api/pagination.md

```diff
@@ -29,18 +29,23 @@ The limit is set to 100 because this is a good trade-off between memory consumpt
 
 To request paginated entities, you can use the [offset method](/documentation/pagination.html#offset-method).
 
-For high volume entities, such as the products, product models, published products and assets, we provide another pagination method, the [search after method](/documentation/pagination.html#search-after-method). We recommend to use this method if you want to have good performances.
+For high volume entities, such as the products, product models, published products, assets and reference entity records, we provide another pagination method, the [search after method](/documentation/pagination.html#search-after-method). We recommend to use this method if you want to have good performances.
 
 ## Search-after method 
 
 To use the search-after method, you have to set the `pagination_type` query parameter to `search_after`. The entities you will get, will then be sorted by product primary key to speed up performance.
 
 :::warning
-This pagination method is only available for products (since 1.7), product models (since 2.0), published products (since 2.0) and assets (since 2.1).
+This pagination method is only available for:
+- products (since 1.7),
+- product models (since 2.0),
+- published products (since 2.0),
+- assets (since 2.1),
+- reference entity records (since 3.0).
 :::
 
-Do note that the `Search after` method is better than `Offset` method in term of performances.
-It is strongly recommended to use it when requesting entities that have high data volume such as the products, the product models, the published products and the assets.
+Do note that the `Search-after` method is better than `Offset` method in term of performances.
+It is strongly recommended to use it when requesting entities that have high data volume such as the products, the product models, the published products, the assets and the reference entity records.
```

Comment:
as we will only provide the search-after with reference entities records, it is not recommended,as it is mandatory. Maybe it can be confusing no?
```
For the reference entities records, as it is the only available method,you don't have to worry about wich method to choose.
```

Maybe you have a better sentence.

Reply:
Yes, I'll try something.

Reply:
Done!